src/common.h.in

```diff
@@ -93,6 +93,9 @@ int pthread_mutex_timedlock(pthread_mutex_t *mutex, const struct timespec *absti
 /** macro for checking session type */
 #define SR_IS_EVENT_SESS(session) (session->ev != SR_SUB_EV_NONE)
 
+/** macro for opening files in SHM */
```

Comment:
Update the description please. Best to mention that all files are created by sysrepo, so there can be no symlinks and forking once a sysrepo connection is created is forbidden.

Reply:
Done. :)